meta/def.rb

```diff
@@ -94,6 +110,7 @@
   mutation 'def foo(_unused = false); end'
   mutation 'def foo(_unused = true); raise; end'
   mutation 'def foo(_unused); end'
+  mutation 'def foo; _unused = true; end'
```

Comment:
@backus Seems very adjacent, and I actually think this one is what I initially wanted. :P Mostly because the other you added is redundant with that one.